lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -59,6 +84,28 @@ def acceptable_pattern?(arg)
             arg.match?(/\[.+=\w+\]/) || arg.match?(/[ >,+]/)
           end
 
+          def specific_matcher_option?(arg, matcher)
+            attribute = arg.slice(/\[[\w\W]*\]/)&.gsub!(/[\[\]]*/, '')
```

Comment:
Cosmetic: does it have to be a bang-`gsub`?
I'm too bad with regexp, do you mind adding a short note what this does exactly?

Reply:
It did not have to be bang-gsub. I have written a description of what I do at regex.